src/utils/retry_utils/src/retry_utils.c

```diff
@@ -259,5 +260,18 @@ void ReadRetryParamsArrayFromAgentConfigJson(
             value = DEFAULT_ENR_REQ_OP_MAX_JITTER_PERCENT;
         }
         params->maxJitterPercent = (double)value;
+
+        //
+        // Get the update polling interval retry setting, or default.
+        //
+        if (!ADUC_JSON_GetUnsignedIntegerField(
+                agentRetrySettingsJsonValue, SETTING_KEY_POLLING_INTERVAL_SECONDS, &context->pollingIntervalInSeconds))
```

Comment:
There might be a case where polling should never be zero to prevent scheduler domination so would make sense to have a separate check for that and logging that information (eg Log_Info("Polling interval in configuration is zero. Minimum polling interval is default value of: %lu", ADUC_DEFAULT_POLLING_INTERVAL_SECONDS) 

Or you could make a definition for the "safe" minimum polling interval (maybe call it 10% of our default ?)  so customers can still make it faster / slower but they're not going to bork everything by making it zero or like 1 or something. 

If you follow Nyquist's sampling law the system should be polling at 3x the rate of the system. So if the lowest delay we have is 60 seconds a recommended sampling/ scheduling window would be 20 s. Just throw math at the problem and it might go away.